ixmp/core.py

```diff
@@ -271,7 +271,21 @@ def add_region_synonym(self, region, mapped_to):
         self._backend.set_node(region, synonym=mapped_to)
 
     def timeslices(self):
-        """Return all sub-annual time slices.
+        """Return all subannual timeslices defined in this Platform instance.
+
+        Timeslices are a way to represent subannual temporal resolution in
+        timeseries data. A timeslice consists of a **name** (e.g., 'january',
+        'summer'), a **category** (e.g., 'months', 'seasons'), and a
+        **duration** given relative to a full year.
+
+        The category and duration do not have any functional relevance within
+        the ixmp framework, but they may be useful for pre- or post-processing.
+        For example, they can be used to filter all timeslices of a certain
+        category (e.g., all months) from the pandas.DataFrame returned by this
+        function or to aggregate subannual data to full-year results.
+
+        See :method:`add_timeslice` to initialize additional timeslices in the
+        Platform instance.
 
         Returns
         -------
```

Comment:
- What are the contents of the returned object?

Reply:
added a line